src/common/util.js

```diff
@@ -33,8 +33,8 @@ export function cmdOrCtrlKey(event) {
   return isMac ? event.metaKey : event.ctrlKey;
 }
 
-export async function showNotification(operation, numMessages, destination, dismissTime = 10000) {
-  let operationAction = browser.i18n.getMessage("operationAction." + operation, ["#num_messages#", "#destination#"]);
+export async function showNotification(operation, numMessages, destination, operationType = "", dismissTime = 10000) {
+  let operationAction = browser.i18n.getMessage("operationAction." + operation + operationType, ["#num_messages#", "#destination#"]);
```

Comment:
Isn't there a dot missing to join between operation and operationType? Reviewing on mobile, sorry if I missed something.

Reply:
Look at background.js line 122, I put the dot there to avoid an `if` in the showNotifications method.

Reply:
This line: https://github.com/micz/quickmove-extension/blob/23451ebe9c69142a59675a84e18ff8c1a51d5547/src/background.js#L122

Reply:
Ah I see. In terms of code separation, background.js shouldn't need to know about this implementation detail in showNotification.

Despite the extra if, I think we should keep it here. If you want to be concise you could do

```javascript
let operationString = `operationAction.${operation}${operationType ? "." + operationType : ""}`;
let operationAction = browser.i18n.getMessage(operationString, ["#num_messages#", "#destination#"]);
```

Reply:
An alternative would be to also add an operationType for the add operation, then we'll always have one.

Reply:
An alternative would be to just keep `operation` as a parameter, and then in the caller instead of passing an extra`operationType` just directly pass `tag.add` and `tag.remove`. This would probably get you the best of both worlds.

Reply:
The last one is the best. I did that one.